pkg/storaged/stratis-details.jsx

```diff
@@ -641,20 +642,40 @@ export const StratisPoolDetails = ({ client, pool }) => {
         return for_each_async(blockdevs, bd => pool.GrowPhysicalDevice(bd.Uuid));
     }
 
-    let alert = null;
+    const alerts = [];
     if (client.features.stratis_grow_blockdevs &&
         blockdevs.some(bd => bd.NewPhysicalSize[0] && Number(bd.NewPhysicalSize[1]) > Number(bd.TotalPhysicalSize))) {
-        alert = (<Alert key="unused-space"
-                        isInline
-                        variant="warning"
-                        title={_("This pool does not use all the space on its block devices.")}>
+        alerts.push(<Alert key="unused-space"
+                           isInline
+                           variant="warning"
+                           title={_("This pool does not use all the space on its block devices.")}>
             {_("Some block devices of this pool have grown in size after the pool was created. The pool can be safely grown to use the newly available space.")}
             <div className="storage_alert_action_buttons">
                 <StorageButton onClick={grow_blockdevs}>{_("Grow the pool to take all space")}</StorageButton>
             </div>
         </Alert>);
     }
 
+    if (pool.AvailableActions !== "fully_functional") {
+        let alert_msg = null;
+        if (pool.AvailableActions === "no_ipc_requests")
+            alert_msg = _("Some actions are unavailable until the issue is resolved manually.");
+        if (pool.AvailableActions === "no_pool_changes")
+            alert_msg = _("Maintainance operations are not possible. Please migrate the data to a new pool.");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18843-20230828-114746-90ccf36f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)